Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -148,6 +148,14 @@ protected theorem IsClique.finsetMap {f : α ↪ β} {s : Finset α} (h : G.IsCl
     (G.map f).IsClique (s.map f) := by
   simpa
 
+/-- If a set of vertices `A` is a clique in subgraph of `G` induced by a superset of `A`,
+ it's embedding is a clique in `G`. -/
+theorem induce_isClique {S : Subgraph G} {F : Set α} {A : Set F} (c : (S.induce F).coe.IsClique A) :
+    G.IsClique (Subtype.val '' A) := by
+  simp_all [Set.Pairwise]
```

Comment:
If simp(_all) is not the tactic that closes the goal, you'll want to squeeze the simp (in this case by using the output from `simp_all?`). This ensures that the rest of the proof remains valid if more simp lemma's are added in the future. Also holds for the other lemmas except `induce_isIndepSet_iff`, because there it is the final call.

Reply:
Yep, looks good to me:)